src/golive/cdn.md

```diff
@@ -52,3 +52,51 @@ When using a CDN the Platform.sh router's HTTP cache becomes redundant.  In most
         # Disable the HTTP cache on this route. It will be handled by the CDN instead.
         enabled: false
 ```
+
+## Secure/hide your project from being accessed directly
+
+When using a CDN, you generally don't want users to access your platform.sh project directly. Luckily, you can secure your connection between you and the CDN with [client authenticated TLS](/configuration/routes/https.html#client-authenticated-tls). 
+
+
+*Note: activating authenticated TLS also means that *you* won't be able to access the project url directly either. If you want to connect to your origin, you'll have to add your own certificate to the configuration as well (see below)*
```

Comment:
I'd prefer different formulation like: Please remember to permit your developers to access the origin by creating your own certificate or white listing the appropriate IP ranges.